exchanges/ftx/ftx_test.go

```diff
@@ -1114,6 +1116,8 @@ func TestGetOrderHistory(t *testing.T) {
 	cp := currency.NewPairWithDelimiter(currency.BTC.String(), currency.USDT.String(), "/")
 	orderReq.Pairs = append(orderReq.Pairs, cp)
 	orderReq.AssetType = asset.Spot
+	orderReq.Side = order.AnySide
+	orderReq.Type = order.AnyType
 	_, err := f.GetOrderHistory(context.Background(), &orderReq)
```

Comment:
I love the use of the validator checks. Could you please add another scenario for `ImmediateOrCancel` to test the expected error? Along with a side of `MissingData` or some other invalid option